corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -177,6 +178,8 @@ def indicators(self):
                 ret.append(make_form_question_indicator(
                     prop['source'], prop['column_id']
                 ))
+            elif prop['type'] == 'case_property' and prop['source'] == 'computed/owner_name':
```

Comment:
seems fine for now but if we keep creeping this we should probably figure out a cleaner strategy for not hard coding a bunch of these